cmd/minishift/cmd/start.go

```diff
@@ -330,6 +330,9 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 		// Should handle cached images
 		//fmt.Println("   Boot ISO:  ", machineConfig.MinikubeISO)
 	}
+
+	cacheMinishiftISO(*machineConfig)
```

Comment:
it is quite uncommon to de-reference this way, also you effectively create a copy now